style.css

```diff
@@ -110,3 +138,238 @@ main {
   margin: 10px;
   list-style: none;
 }
+
+.snapshoot {
+  width: inherit;
+  height: 220px;
+  display: block;
+}
+
+.snapshoot > img {
+  width: 100%;
+  height: 100%;
+}
+
+.frame {
+  width: 236px;
+  height: 32px;
+  display: grid;
+  margin: 10px 0;
+  align-items: center;
+  grid-template-columns: auto auto auto auto auto;
+}
+
+.frame > p,
+.frame > h2 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.frame > h2 {
+  color: #344563;
+}
+
+.frame > div {
+  background-color: #e5e5e5;
+  border-radius: 50%;
+  height: 8px;
+  width: 8px;
+}
+
+.text > p {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
+
+.lang {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-column-gap: 8px;
+  width: 203px;
+  height: 24px;
+  margin: 20px 0 5px 0;
+}
+
+.lang > p {
+  height: inherit;
+  display: grid;
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  align-items: center;
+  text-align: center;
+}
+
+button {
+  padding: 10px;
+  height: 48px;
+  top: calc(50% - 48px / 2);
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  margin: 20px 0;
+}
+
+.lan-tle {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 2px solid #f7f7f9;
+}
+
+.toggle {
+  display: grid;
+  grid-template-rows: 66px 66px 66px;
+  grid-row-gap: 10px;
+  margin: 10px 0;
+}
+
+.tags {
+  border-bottom: none;
+}
+
+.container-lan {
+  display: grid;
+  grid-template-rows: auto auto auto;
+  width: 327px;
+}
+
+.lan-tle > h2 {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  height: 72px;
+  display: flex;
+  align-items: center;
+}
+
+.lan-tle > i {
+  font-weight: 500;
+  font-size: 27px;
+  color: #5e6c84;
+}
+
+.lan {
+  display: grid;
+  padding: 0 10px;
+  grid-template-columns: 62px auto;
+  background-color: #f7f7f9;
+  border-radius: 8px;
+  align-items: center;
+}
+
+.lan > p {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
```

Comment:
- [ ] As mentioned before, you should consider changing your `font-family`, to match the Figma design better.